roles/extract_operator_bundle/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+- name: "Ensure that the operator bundle directory is empty"
+  file:
+    state: "absent"
+    path: "{{ operator_bundle_dir }}"
+  when: operator_bundle_dir is defined
+
+- name: "Ensure that the operator testing directory exists and is empty"
+  file:
+    state: "{{ item }}"
+    path: "{{ operator_work_dir }}"
+  with_items:
+    - absent
+    - directory
+  when: operator_work_dir is defined
+
+- name: "Copy the bundle image layers into a local directory using skopeo"
+  shell: "skopeo copy docker://{{ bundle_image }} dir:{{ operator_bundle_dir }} -f oci"
+  register: skopeo_copy_result
+
+- name: "Check that the image manifest.json file exists"
+  stat:
+    path: "{{ operator_bundle_dir }}/manifest.json"
+  register: image_manifest_stat_result
```

Comment:
What is a purpose of this check?

The image was copied from real docker registry and not tarball. And thus is has all necessary files which are expected from valid docker image.

Reply:
Removed along with other changes.